pyclient/pymtt.py

```diff
@@ -28,7 +28,14 @@
 #    sys.exit('MTT usage error: add -h for help')
 
 # define the cmd line arguments
-parser = argparse.ArgumentParser(description="usage: %prog [options] testfile1 testfile2 ...")
```

Comment:
Why remove the old description of MTT usage, instead of adding on to the description?

Reply:
because its redundant with the usage that argparse provides